src/frontend/komponenter/Visningskomponenter/Alerts.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { forwardRef } from 'react';
+
+import { Alert, AlertProps } from '@navikt/ds-react';
+
+export const AlertError = forwardRef<HTMLDivElement, Omit<AlertProps, 'variant'>>((props, ref) => {
+    return <Alert variant={'error'} {...props} ref={ref} />;
+});
+
+export const AlertSuccess = forwardRef<HTMLDivElement, Omit<AlertProps, 'variant'>>(
+    (props, ref) => {
+        return <Alert variant={'success'} {...props} ref={ref} />;
+    }
+);
+
+export const AlertInfo = forwardRef<HTMLDivElement, Omit<AlertProps, 'variant'>>((props, ref) => {
+    return <Alert variant={'info'} {...props} ref={ref} />;
+});
+export const AlertWarning = forwardRef<HTMLDivElement, Omit<AlertProps, 'variant'>>(
+    (props, ref) => {
+        return <Alert variant={'warning'} {...props} ref={ref} />;
+    }
+);
```

Comment:
Er det noen grunn til å lage disse? 🤔 Så lenge komponentene skal være like seg jeg ikke noen grunn til å lage egne kopier av Aksel. Det gir oss bare en ting mer å holde oppdatert 🤓

Reply:
Det blir ju som en wrapper, og enklere bruk.
Hvis aksel oppdaterer, trenger vi kun å oppdatere her, i stedet for på hvert sted `<Alert ...` er i bruk ? 
Føler også det er enklere og kortere å skrive `<AlertSuccess` enn `<Alert variant=´success´`
Så får man vel også wrappet denne med styled

`const styledAlertWarning = styled(AlertWarning)`

Reply:
Tror jeg foretrekker å bruke komponetene fra Aksel direkte, så slipper jeg å "lære meg" hva wrapper-komponenten faktisk gjør. Tror disse ble inført på på EF før Aksel hadde en god løsning med alt vi trengte 🤔 

For å kunne bruke styled direkte uten wrapper kan du ta `styled(Alert).attrs({variant: 'info'})` 🤠 

@sarahjelle  har sikkert litt tanker om vi skal bruke wrappers eller ikke.

Reply:
Ser fordelen dersom Aksel oppdateres, men vi har fjernet dette generelt i koden. Syntes det er cleanere at man bruker Aksel direkte slik at man vet hvordan komponenten fungerer og ikke må forholde seg til enda en komponent. 

Til nå har vi kun laget wrappers dersom det faktisk gjør noe mer. F.eks. for inputfelter hvor vi skal ha lesevisning.

Dersom Aksel oppdaterer seg er det mer sannsynlig at det lages en ny komponent med nytt navn. Da bør ikke vår komponent hete `AlertSuccess` heller lenger og det må oppdateres flere steder. 

Annen fordel er at imports reflekterer at vi faktisk bruker akselkomponentene. Ved første øyekast ser man ikke at dette er en komponent fra Aksel